examples/todoapp/buildSrc/buildSrc/src/main/kotlin/Deps.kt

```diff
@@ -12,7 +12,11 @@ fun initDeps(project: Project) {
 object Deps {
     object JetBrains {
         object Kotlin {
-            private val VERSION get() = properties["kotlin.version"]
+            //todo remove, when updated to new Compose with support Kotlin 1.6.20 or newer
+            private val XCODE_WORKAROUND_KOTLIN_VERSION = "1.6.20"
+            private val workaroundIsBuildWithXcode = System.getenv("XCODE_VERSION_ACTUAL") != null
+
+            private val VERSION get() = if (workaroundIsBuildWithXcode) XCODE_WORKAROUND_KOTLIN_VERSION else properties["kotlin.version"]
```

Comment:
Let's don't check XCODE_VERSION_ACTUAL, just override Kotlin version without conditions:
```
private val VERSION get() = "1.6.20"
```
Theoretically, some tools don't set this variable. If that happen, we build our project with two Kotlin versions, which isn't good for stability.

Also, please add TODO to gradle.properties, so we don't forget about this, when we upgrade Kotlin version to 1.6.20 for the whole project.

Reply:
XCODE_VERSION_ACTUAL is environment variable inside of Xcode.
So I just check if this env exists, then we need to set version hardcoded version 1.6.20 (Because build inside Xcode).
In other cases we need to use version from gradle.properties (1.6.10)

I will remove this workaround after stable version of Compose with Kotlin 1.6.20 or newer

Reply:
> In other cases we need to use version from gradle.properties (1.6.10)

I get it now, that this variable is also used for building desktop/android projects. But my concern is only related to iOs. Imagine, we develop in Idea, where is a similar to XCODE_VERSION_ACTUAL variable exists (` System.getProperty("idea.active")`. We run from Idea, and the project uses Kotlin 1.6.20. All works fine. We make a commit, and push to CI. CI doesn't build with IDEA, so it uses Kotlin 1.6.10. If something isn't backward compatible, it will fail. 

Can we create a copy of this block, and name it something like KOLTIN_IOS?

Reply:
We change this behavior. Moves to Xcode shell script -Pkotlin.version=1.6.20

Reply:
@treitter Can you please review and merge this PR to your fork ?